docs/reference/commandline/events.md

```diff
@@ -31,7 +31,12 @@ Options:
 ## Description
 
 Use `docker events` to get real-time events from the server. These events differ
-per Docker object type.
+per Docker object type. Different event types have different scopes. The local 
+scoped events are only seen on the node they take place on, and swarm scoped 
+events are seen on all managers.
+
+Note that there is a limited backlog of events you can filter. `docker events` is 
```

Comment:
If this is a "note", this should use our standard markup for notes:

```Markdown
> **Note**: There is a limited backlog of events you can filter. `docker events`
> is  a rolling log, and has at most 1000 events at a time.
```

However, trying to preserve "notes" for exceptional cases; perhaps this can be rephrased slightly to not be a "note". Something like:

```Markdown
Docker uses a rolling log to store the last 1000 events, which means that at
most 1000 events at a time are returned.
```

(not entirely happy with that example; perhaps you have a better suggestion)

Reply:
How about just "Only the last 1000 log events are returned. You can use filters to further limit the number of events returned."